app/models/user.rb

```diff
@@ -114,7 +114,7 @@ def get_an_identity_user
   end
 
   def ecf_user
-    return if ecf_id.blank? || Rails.application.config.npq_separation[:ecf_api_disabled]
+    return if ecf_id.blank? || Feature.ecf_api_disabled?
```

Comment:
I think we'll also need to disable the following as they call this method to avoid errors:
```
GetAnIdentityIdSyncJob
EcfUserUpdater
```